lib/polkadot_js_service/src/utils/costom_types.js

```diff
@@ -0,0 +1,209 @@
+const costomTypes = {
```

Comment:
instead of duplicating https://github.com/DataHighway-DHX/node/blob/master/custom_types.json, could you read from the existing file instead please so we don't have to remember to update and maintain the custom types in two places